app/javascript/dashboard/routes/dashboard/settings/inbox/components/AccountHealth.vue

```diff
@@ -0,0 +1,226 @@
+<script setup>
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+import ButtonV4 from 'next/button/Button.vue';
+
+const props = defineProps({
+  healthData: {
+    type: Object,
+    default: null,
+  },
+});
+
+const { t } = useI18n();
+
+const QUALITY_COLORS = {
+  GREEN: 'text-n-teal-11',
+  YELLOW: 'text-n-amber-11',
+  RED: 'text-n-ruby-11',
+  UNKNOWN: 'text-n-slate-12',
+};
+
+const STATUS_COLORS = {
+  APPROVED: 'text-n-teal-11',
+  PENDING_REVIEW: 'text-n-amber-11',
+  AVAILABLE_WITHOUT_REVIEW: 'text-n-teal-11',
+  REJECTED: 'text-n-ruby-9',
+  DECLINED: 'text-n-ruby-9',
+};
+
+const MODE_COLORS = {
+  LIVE: 'text-n-teal-11',
+  SANDBOX: 'text-n-slate-11',
+};
+
+const healthItems = computed(() => {
+  if (!props.healthData) {
+    return [];
+  }
+
+  const {
+    display_phone_number: displayPhoneNumber,
+    verified_name: verifiedName,
+    name_status: nameStatus,
+    quality_rating: qualityRating,
+    messaging_limit_tier: messagingLimitTier,
+    account_mode: accountMode,
+  } = props.healthData;
+
+  return [
+    {
+      key: 'DISPLAY_PHONE_NUMBER',
```

Comment:
Do we need to keep `key: 'DISPLAY_PHONE_NUMBER'` in uppercase? Keys are usually `camelCase` or `snake_case` (e.g., `displayPhoneNumber` / `display_phone_number`) rather than uppercase.